os/pm/Kconfig

```diff
@@ -82,6 +82,14 @@ config PM_MEMORY
 
 		Setting CONFIG_PM_MEMORY=1 disables all smoothing.
 
+if ARCH_CORTEXA32
+config PM_DVFS
+	bool "Dynamic Voltage Frequency Scaling"
+	default y
```

Comment:
This shouldnt be enabled by default and dependency on CORTEXA32 shouldnt be added in common code.  

This should be selected by CORTEXA32 (or I think should be selected by the board RTL8730E). @amogh-samsung could you please check this?

Reply:
Do you suggest that selecting 8730E and PM should automatically select CONFIG_PM_DVFS?

Reply:
Yes, if possible that would be good.

Reply:
As suggested, moved to definition under os/arch/arm/Kconfig ARCH_CHIP_AMEBASMART.